detector_parameters:
  adaptiveThreshWinSizeMin: 3
  adaptiveThreshWinSizeMax: 32
  adaptiveThreshWinSizeStep: 5
  adaptiveThreshConstant: 5
  minMarkerPerimeterRate: 0.06
  maxMarkerPerimeterRate: 3
  polygonalApproxAccuracyRate: 0.04
  minCornerDistanceRate: 0.05
  minDistanceToBorder: 3
  minMarkerDistanceRate: 0.05
# Corner refinement method: 0 for CORNER_REFINE_NONE, 1 for CORNER_REFINE_SUBPIX,
# 2 for CORNER_REFINE_CONTOUR, 3 for CONTOUR_REFINE_APRILTAG
  cornerRefinementMethod: 2
  cornerRefinementWinSize: 5
  cornerRefinementMaxIterations: 30
  cornerRefinementMinAccuracy: 0.1
  markerBorderBits: 1
  perspectiveRemovePixelPerCell: 4
  perspectiveRemoveIgnoredMarginPerCell: 0.13
  maxErroneousBitsInBorderRate: 0.35
  minOtsuStdDev: 4.0
  errorCorrectionRate: 0.6
